packages/package.json

```diff
@@ -18,8 +18,8 @@
     "lint-fix": "lerna run lint-fix",
     "prettier-fix": "lerna run prettier-fix",
     "test": "lerna run test",
-    "publish": "npm run build && lerna publish",
-    "force-publish": "lerna run tsc && lerna publish --force-publish --no-git-tag-version --no-push"
+    "publish:lerna": "npm run build && lerna publish",
+    "force-publish": "lerna run tsc && lerna publish --force-publish --no-git-tag-version --no-push --ignore-changes '**'"
```

Comment:
Let's have the actual command we use so we can use scripts and have stuff done consistently amongst developers